### ISSUE TITLE
Add ignore: ["comments"] to block-no-empty

### ISSUE THREAD

**Issue by saschask** [stylelint/stylelint#3925]
> Clearly describe the bug

If rule `block-no-empty` is enabled, no error is shown in case a block contains comments-only. Of course a comment can be seen as some kind of content but finally the block will do nothing styling-wise which is why it should be considered as empty.

As an alternative an option could be introduced in order to define whether the rule should show an error in case of a completely empty block or already in case of a block with comments-only.

> Which rule, if any, is the bug related to?

`block-no-empty`

> What CSS is needed to reproduce the bug?

```css
a {
  /* this block with `a` selector should be considered as empty although a comment is there */
}
```

> What stylelint configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "block-no-empty": true
  }
}
```

> Which version of stylelint are you using?

`9.10.0`

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

CLI with `stylelint "source/**/*.css"`

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

No.

> What did you expect to happen?

An error should be shown.

> What actually happened (e.g. what warnings or errors did you get)?

No error is shown.

**Comment by evilebottnawi** [comment]
I think it is normal and expected, eslint also doesn't consider block with only comments as empty, maybe we can implement option `ignoreComments`

**Comment by saschask** [comment]
Such a new option would be awesome. Thank you in advance!

**Comment by jeddy3** [comment]
Sounds good to me.

@saschask If you'd like to contribute this option, there's [a section in the Developer Guide](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/rules.md#adding-an-option-to-an-existing-rule) on how to get started.

The option should be called `ignore: ["comments"]` to adhere to the [conventions](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/rules.md#keyword-ignore-and-except). Have a look at another rules `ignore` option for a blue print on code, docs and tests.
